test_reproducibility.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import Path
+import tempfile
+
+import pandas as pd
+from misc.utils import set_seed
+from runners import main_runner
+
+
+TCASE_CONFIG = {
```

Comment:
Define these resources as `pytest` fixtures

Reply:
yes, I know pytest fixtures, but we're not developing here a code that is supposed to have 100% coverage. For me it's clean enough